target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -18,12 +18,19 @@ ath10kcal_extract() {
 	local part=$1
 	local offset=$2
 	local count=$3
-	local mtd
+	local mtd size1 size2
```

Comment:
I prefer variable name which is more descriptive, maybe `cal_size`?
(why just one, see below)